auto_nag/scripts/workflow/no_severity.py

```diff
@@ -226,30 +226,38 @@ def get_bz_params(self, date):
                     "f5": "product",
                     "o5": "changedafter",
                     "v5": second,
-                    "f6": "CP",
-                    "j7": "AND",
-                    "f7": "OP",
-                    "f8": "component",
-                    "o8": "changedbefore",
-                    "v8": second,
-                    "n9": 1,
+                    "n6": 1,
+                    "f6": "component",
+                    "o6": "changedafter",
+                    "v6": second,
+                    "f7": "CP",
+                    "j8": "AND",
+                    "f8": "OP",
                     "f9": "component",
-                    "o9": "changedafter",
+                    "o9": "changedbefore",
+                    "n10": 1,
+                    "f10": "product",
+                    "o10": "changedafter",
+                    "v10": second,
                     "v9": second,
```

Comment:
could you move this line to after the "o9" line so the "9" are close together?

Reply:
Done in a56698a3c414f7eb2ba0b2b6f739ad8057fec302. It was by mistake.